products/idn/api/patch-guide.md

```diff
@@ -0,0 +1,474 @@
+  }
+]
+```
+
+This request specifies the top-level path, "connectorAttributes", before following it with the path where the value is going to be removed from, "filterString". 
+
+## Specify a value 
+
+For some of the operations, once you have specified the operation you want to apply and the path you want to apply it to, you need to specify the value you want to send with the request. The value you specify must fit the path's data type. 
+
+You can specify a single simple value for an operation. In this example from earlier, the PATCH request replace's the source's description: 
+
+```text
+PATCH https://{tenant}.api.identitynow.com/v3/sources/:id
+```
+
+```json
+[
+  {
+    "op": "replace",
+    "path": "/description",
+    "value": "new description"
+  }
+]
+```
+
+You can also specify multiple values for an operation to be applied to. For example, this PATCH request replaces the source's current features with a number of new ones, all in the same path: 
+
+```text
+PATCH https://{tenant}.api.identitynow.com/v3/sources/:id
+```
+
+```json
+[
+  {
+    "op": "replace",
+    "path": "/features",
+    "value": [
+      "PASSWORD",
+      "PROVISIONING",
+      "ENABLE",
+      "AUTHENTICATE"
+    ]
+  }
+]
+```
+
+A value can also be an object that contains other values within it. For example, this PATCH request adds a new "location" attribute to the end of the array of a source's schema's attributes: 
+
+```text
+PATCH https://{tenant}.api.identitynow.com//v3/sources/:sourceId/schemas/:schemaId
+```
+
+```json
+[
+  {
+    "op": "add",
+    "path": "/attributes/-",
+    "value": {
+      "name": "location",
+      "type": "STRING",
+      "schema": null,
+      "description": "Employee location",
+      "isMulti": false,
+      "isEntitlement": false,
+      "isGroup": false
+    }
+  }
+]
+```
+
+This request uses the [PATCH Source Schema](https://developer.sailpoint.com/idn/api/v3/update-source-schema) endpoint to add a new attribute, along with its details, to the end of the array of a source's schema's attributes. 
+
+This example uses the "-" after the path to indicate that the value will be added to the end of the array. You can specify the position in the array "/" following the path. For example, if there were a list of 3 attributes, you could specify "/attributes/2" and the request would add the new "location" attribute to the second position in the array. 
+
+## Specify a from 
+
+For "move" and "copy" operations, you don't necessarily have to specify a "value" to apply the operations to, but you must specify a "from", a JSON Pointer representing the location you are moving or copying the value from. 
+
+## Send the PATCH request 
+
+Once you have specified the operation you want to apply, the path you want to apply it to, and the changes you want to make with that operation, you can send the PATCH request. 
+
+When the request is successful, the updated resource is returned. 
```

Comment:
You'll want a section in this guide that tells users they need to apply a special content type header for PATCH requests: `Content-Type: application/json-patch+json`.

Reply:
Will do.